<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>163</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBoxProvisions">
       <property name="title">
        <string>Provisions</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="rButtonPasta">
          <property name="text">
           <string>Paste</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rButtonRice">
          <property name="text">
           <string>Rice</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rButtonFlour">
          <property name="text">
           <string>Flour</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxFruit">
       <property name="title">
        <string>Fruit</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QRadioButton" name="rButtonApple">
          <property name="text">
           <string>Apple</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rButtonBanana">
          <property name="text">
           <string>Banana</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rButtonLemon">
          <property name="text">
           <string>Lemon</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxKg">
       <property name="title">
        <string>Kg</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QRadioButton" name="rButtonOne">
          <property name="text">
           <string>One Kg</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rButtonTwo">
          <property name="text">
           <string>Two Kg</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rButtonThree">
          <property name="text">
           <string>Three Kg</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxSecure">
       <property name="title">
        <string>Corona Secure</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QRadioButton" name="rButtonGlove">
          <property name="text">
           <string>Glove</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rButtonMask">
          <property name="text">
           <string>Mask</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rButtonCologne">
          <property name="text">
           <string>Cologne</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonOk">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
